dockstore-integration-testing/src/test/java/io/dockstore/client/cli/DAGWorkflowTestIT.java

```diff
@@ -349,7 +349,7 @@ public void testComplexImportWdlWorkflow() throws ApiException {
 
     @Test
     public void testReallyComplexImportedWdlWorkflow() throws ApiException {
-        final List<String> strings = getJSON("dockstore-testing/gatk-sv-clinical", "/GATKSVPipelineClinical.wdl", "wdl", "dockstore-test");
+        final List<String> strings = getJSON("dockstore-testing/gatk-sv-clinical", "/GATKSVPipelineClinical.wdl", "wdl", "master");
```

Comment:
I suggest creating a tag/release and referencing the tag from here -- in case we ever make additional changes or pull to master from the repo we forked from.